ui/apps/ui/src/app/collections/data/all/filters.data.ts

```diff
@@ -28,6 +22,12 @@ export const allCollectionsFilters: IFiltersConfig = {
       label: 'Scientific discipline',
       type: 'multiselect',
     },
+    {
+      id: 'unified_categories',
+      filter: 'unified_categories',
+      label: 'Research step',
+      type: 'multiselect',
+    },
```

Comment:
Why have you moved the filter?

Reply:
just wanted to change order of filters according to project:

https://www.figma.com/file/D2vgx66m1ADdBY0A46NdZ6/Search-Service?node-id=171%3A2204&t=ew28qVM7iJdnPjyX-1